### PR TITLE
implement new design for browser wallet

### PR DESCRIPTION
**Pull request by krzkaczor**
image
image

### CODE REVIEW

File: app/components/HorizontalLine.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as cn from "classnames";
```
Comment (banciur) [inline app/components/HorizontalLine.tsx]: Maybe here we should just restyle `<hr />` element? and put it as global style

File: app/components/walletSelector/WalletBrowser.module.scss [hunk 2]
```diff
@@ -9,3 +9,9 @@
   margin-top: .9375rem;
```
Comment (banciur) [inline app/components/walletSelector/WalletBrowser.module.scss]: we could change it to `$spacer`

File: app/components/walletSelector/WalletBrowser.module.scss [hunk 3]
```diff
@@ -9,3 +9,9 @@
...
   margin-bottom: .9375rem;
 }
+
+.wallet-logos {
+  img {
+    height: 85px;
```
Comment (banciur) [inline app/components/walletSelector/WalletBrowser.module.scss]: Aren't we are using `rem` units?

File: app/styles/variables.scss [hunk 4]
```diff
@@ -8,7 +8,16 @@ $gray-almost-dark: #7d919b;
...
 $white: #fff;
+$bright-orange: #ff6d00;
+
 
 $smaller-font-size: .8125rem;
 
 $shadow:  0 .125rem .25rem 0 rgba(0, 0, 0, .1);
+
+
+$headings-font-family: "Lato", sans-serif;
```
Comment (banciur) [inline app/styles/variables.scss]: If those are bootstrap variable overrides I would move them to `neufund-theme.scss` where already are some overrides. Or if this place is better I would move ones from there here,

File: app/components/walletSelector/WalletBrowser.module.scss (hunk 2 above)
Comment (banciur) [reply app/components/walletSelector/WalletBrowser.module.scss]: Or use (spacing utils)[https://getbootstrap.com/docs/4.0/utilities/spacing/] sadly those are not implemented in react-strap

File: app/components/HorizontalLine.tsx (hunk 1 above)
Comment (krzkaczor) [reply app/components/HorizontalLine.tsx]: yeah, i thought about this but it seems like styling HR is not so straight forward.

File: app/styles/variables.scss (hunk 4 above)
Comment (krzkaczor) [reply app/styles/variables.scss]: I thought that `variables.scss` is the file that we put you know, all variables :D I hope we won't have many other variables then bootstrap overrides.

**Pull request merged**